website/templates/profile/personal_tokens_list.mako

```diff
@@ -0,0 +1,70 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Personal Access Tokens</%def>
+<%def name="content()">
+<h2 class="page-header">Settings</h2>
+
+<div id="personalTokenListPage" class="row">
+    <div class="col-sm-3 affix-parent">
+      <%include file="include/profile/settings_navpanel.mako" args="current_page='personal_tokens'"/>
+    </div>
+
+    <div class="col-sm-9 col-md-7">
+
+        <div id="tokensList" class="panel panel-default scripted" style="display:none;" data-bind="visible: true">
+            <div class="panel-heading clearfix">
+                <h3 class="panel-title" style="padding-bottom: 5px; padding-top: 5px;">Personal Access Tokens</h3>
+                <div class="pull-right">
+                    <a data-bind="attr: {href: webCreateUrl}" role="button" class="btn btn-sm btn-default">New token</a>
+                </div>
+            </div>
+            <div class="panel-body">
+
+                <p> Personal access tokens function like ordinary OAuth access tokens. They can be used to authenticate to the API.</p>
+
+                <p data-bind="visible: (tokenData().length == 0)">You have not created any access tokens that can connect to the OSF.</p>
+                <div id="if-tokens" data-bind="visible: (tokenData().length > 0)">
+                    <p>You have registered the following access tokens that can connect to the OSF:</p>
```

Comment:
This language worked for applications, but less so for tokens.

Per https://github.com/settings/tokens , something like the following may be better:
> "You have generated the following personal access tokens:"